system-test/datastore.ts

```diff
@@ -308,6 +308,39 @@ describe('Datastore', () => {
       await datastore.delete(postKey);
     });
 
+    // TODO: Modify this
+    it('should save/get/delete from a snapshot', async () => {
+      function sleep(ms: number) {
+        return new Promise(resolve => setTimeout(resolve, ms));
+      }
+      const post2 = {
+        title: 'Another way to make pizza',
+        tags: ['pizza', 'grill'],
+        publishedAt: new Date(),
+        author: 'Silvano',
+        isDraft: false,
+        wordCount: 400,
+        rating: 5.0,
+        likes: null,
+        metadata: {
+          views: 100,
+        },
+      };
+      const path = ['Post', 'post1'];
+      const postKey = datastore.key(path);
+      await datastore.save({key: postKey, data: post});
+      await sleep(1000);
+      const savedTime = Date.now();
+      await sleep(1000);
+      // Save new post2 data, but then verify the timestamp read has post1 data
+      await datastore.save({key: postKey, data: post2});
+      const [entity] = await datastore.get(postKey, {readTime: savedTime}); // Include new options here
```

Comment:
maybe it would be nice to do two of these; one for each post

Reply:
Good plan.

Reply:
Done